components/installer/pkg/config/installation-data_test.go

```diff
@@ -3,26 +3,27 @@ package config
 import (
 	"testing"
 
+	"github.com/kyma-project/kyma/components/installer/pkg/apis/installer/v1alpha1"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestConvertTopMap(t *testing.T) {
 
 	Convey("The function consumes a string of comma-separated values and converts it to a string-struct map", t, func() {
 
-		var fakeComponentList string
-
 		Convey("Should the string be empty, the map must not contain any keys", func() {
 
-			fakeComponentList = ""
+			fakeComponentList := []v1alpha1.KymaComponent{}
 			testMap := convertToMap(fakeComponentList)
 
 			So(len(testMap), ShouldEqual, 0)
 		})
 
 		Convey("Should the string contain just one element, the map must be of length 1 as well", func() {
 
-			fakeComponentList = "core"
+			fakeComponentList := []v1alpha1.KymaComponent{
```

Comment:
you need to update test name, because now it is not a string. Please check other cases.

Reply:
True, fixed